introduction/examples/thompson/friends.py

```diff
@@ -0,0 +1,27 @@
+# zadanie 1
+friends = {"John": "polite", "Andrew": "kind", "Suzy": "wise", "Claire": "joyful", "Brian": "kind"}
+
+# zadanie 2
+friends_names = list(friends.keys())
+special_trades_list = list(set(friends.values()))
```

Comment:
Takie rozwiązania to tzw _Pythonic solution_ bardzo eleganckie i proste rozwiązania danych zaganień. Świetne!